src/index.js

```diff
@@ -24,9 +25,19 @@ export class Wizard extends Component {
     Local Properties
   \***************************************************************************/
 
+  static defaultProps = {
```

Comment:
I usually stick `defaultProps` and `propTypes` down into their own section at the bottom of a class as they serve a specific purpose outside of the functional code.